lib/node_modules/@stdlib/blas/ext/base/wasm/dasumpw/docs/types/test.ts

```diff
@@ -0,0 +1,409 @@
+	dasumpw.ndarray( x.length, x, undefined, 0 ); // $ExpectError
+	dasumpw.ndarray( x.length, x, [], 0 ); // $ExpectError
+	dasumpw.ndarray( x.length, x, {}, 0 ); // $ExpectError
+	dasumpw.ndarray( x.length, x, ( x: number ): number => x, 0 ); // $ExpectError
+}
+
+// The compiler throws an error if the `ndarray` method is provided a fifth argument which is not a number...
+{
+	const x = new Float64Array( 10 );
+
+	dasumpw.ndarray( x.length, x, 1, '10' ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, true ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, false ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, null ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, undefined ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, [] ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, {} ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, ( x: number ): number => x ); // $ExpectError
+}
+
+// The compiler throws an error if the `ndarray` method is provided an unsupported number of arguments...
+{
+	const x = new Float64Array( 10 );
+
+	dasumpw.ndarray(); // $ExpectError
+	dasumpw.ndarray( x.length ); // $ExpectError
+	dasumpw.ndarray( x.length, x ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1 ); // $ExpectError
+	dasumpw.ndarray( x.length, x, 1, 0, 10, 2 ); // $ExpectError
+}
+
+// Attached to the main export is a `Module` constructor which returns a module...
+{
+	const mem = new Memory({
+		'initial': 0
+	});
+
+	dasumpw.Module( mem ); // $ExpectType Module
+}
+
+// The compiler throws an error if the `Module` constructor is not provided a WebAssembly memory instance...
+{
+	dasumpw.Module( '10' ); // $ExpectError
+	dasumpw.Module( true ); // $ExpectError
+	dasumpw.Module( false ); // $ExpectError
+	dasumpw.Module( null ); // $ExpectError
+	dasumpw.Module( undefined ); // $ExpectError
+	dasumpw.Module( [] ); // $ExpectError
+	dasumpw.Module( {} ); // $ExpectError
+	dasumpw.Module( ( x: number ): number => x ); // $ExpectError
+}
+
+// The `Module` constructor returns a module instance having a `main` method which returns a number...
+{
+	const mem = new Memory({
+		'initial': 1
+	});
+	const mod = dasumpw.Module( mem );
+
+	mod.main( 10, 0, 1 ); // $ExpectType number
+}
+
+// The compiler throws an error if the `main` method of a module instance is provided a first argument which is not a number...
+{
+	const mem = new Memory({
+		'initial': 1
+	});
+	const mod = dasumpw.Module( mem );
+
+	mod.main( '10', 0, 1 ); // $ExpectError
+	mod.main( true, 0, 1 ); // $ExpectError
+	mod.main( false, 0, 1 ); // $ExpectError
+	mod.main( null, 0, 1 ); // $ExpectError
+	mod.main( undefined, 0, 1 ); // $ExpectError
+	mod.main( [], 0, 1 ); // $ExpectError
+	mod.main( {}, 0, 1 ); // $ExpectError
+	mod.main( ( x: number ): number => x, 0, 1 ); // $ExpectError
+}
+
+// The compiler throws an error if the `main` method of a module instance is provided a second argument which is not a number...
```

Comment:
Same type of mistake. Please go back through this file and remove the various copy-paste bugs.